inst/extdata/emissions/Bond_BC_1990.csv

```diff
@@ -0,0 +1 @@
RailTransport_Diesel,5.24,1.01,1.55,0.15,0.28,5.92,0.42,1.04,0.74,0.18,0.14,0.01,0.1,0.03,0,0.26,0.33
RailTransport_Biomass,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0
OtherTransport_Coal,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
OtherTransport_Gas,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
OtherTransport_Petroleum,0,0,0.27,0,0.08,0.75,0,0.04,0.02,0,0,0,0,0.03,0,0,0
OtherTransport_Gasoline,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
OtherTransport_Diesel,3.08,0.02,0.04,0,1.25,14.88,0.14,0,0.04,0,0,0,0,1.89,0,0,0
ShipTransport_Coal,0,0,0,0,0.17,0,0.01,0,0,0,0,0,0,0,0,0,0
ShipTransport_Gas,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
ShipTransport_Petroleum,0.35,0.68,1.16,2.73,0.45,1.84,0.4,0.57,0.31,0.09,1.19,0,0.01,0.24,0.01,0.82,0
ShipTransport_Gasoline,0.14,0,0.02,0,0,0,0,0,0,0,0,0,0,0,0,0,0
ShipTransport_Diesel,0,0.83,5.69,1.5,0.22,7.66,0.07,2.18,0.27,1.44,0.46,0,0.11,0.49,0,0.32,0
InterBunkers_Coal,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
InterBunkers_Gas,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
InterBunkers_Petroleum,22.82,0.72,28.29,5.14,0.75,4.7,0.35,2.54,0.11,12.01,1.39,9.33,4.49,3.03,0.45,0.42,0.55
InterBunkers_Gasoline,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
InterBunkers_Diesel,7.16,0.23,8.36,0.2,0.26,0,0.15,0.42,0.04,1.04,0.27,0.32,1.04,0.73,0.28,0.14,0.12
CommBuildings_Coal,19.92,0.04,4.59,0.26,0.56,77.46,11.16,22.87,15.6,0,0.12,0,5.87,0,0,0,0
CommBuildings_Gas,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
CommBuildings_Petroleum,0.28,0.03,0.13,0.86,0,0.12,0.01,0.17,0,0.01,0.31,0.14,0,0.02,0,0.01,0.03
CommBuildings_Gasoline,0.29,0,0,0,0,0,0,0,0,0,0,0,0,0.01,0,0,0
CommBuildings_Diesel,0.7,0.2,1.42,0.62,0.01,0,0.01,8.48,0,1.06,0.14,0.03,1.14,0.85,0.59,0.2,0.11
CommBuildings_Biomass,3.82,0,0.03,0,0,0,0.04,0,0,0.12,0,0,0.86,0.08,0,0.25,0
Residential_Coal,12.45,0.46,18.79,0,0.33,0,35.81,322.39,27.66,0.11,2.7,0,8.17,0.78,0,0,0
Residential_Gas,0.01,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
Residential_Petroleum,0.48,0.07,1.01,1.29,0.03,0.43,0.07,0.25,1.12,1.15,0.08,1.16,0.88,0.5,0.12,0.32,0.41
Residential_Gasoline,0,0,0.03,0,0,0,0,0,0,0,0,0,0,0.02,0,0,0
Residential_Diesel,1.2,0.23,2.99,0,0,0,0.05,0.05,3.16,1.06,0,0.08,0.22,0.29,0,0,0
Residential_Biomass,47.96,6.33,39.02,0.02,3.41,0,7.39,430.42,277.53,192.73,0,0.07,222.99,32.44,0.42,14.5,5.47
UrbanBurn_OpenWaste,2.07,0.24,2.11,0.87,0.13,2.2,0.44,11.87,5.24,3.87,0.35,0.96,6.78,1.69,0.3,1.19,0.57
UrbanBurn_OpenBiomass,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
RuralBurn_OpenWaste,0.69,0.08,1.02,0,0,0.72,0.42,0,0,0,0,0,0,0,0,0,0
RuralBurn_OpenBiomass,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
SavannaBurn_OpenWaste,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
SavannaBurn_OpenBiomass,0,0,0.84,0,129.31,0.01,0,48.15,4.11,18.34,0,0.01,1121.92,124.83,4.33,262.8,0
ForestBurn_OpenWaste,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
ForestBurn_OpenBiomass,57.49,40.03,42.28,0.37,13.6,81.21,1.22,47.66,24.71,153.03,0.05,3.16,293.02,205.72,4.86,237.12,32.65
AgResidueBurn_OpenWaste,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
AgResidueBurn_OpenBiomass,3.02,12.25,10.72,1.28,20.7,16.32,4.32,78.49,57.72,34.03,1.19,2.93,50.62,11.98,5.58,14.05,2.37
```

Comment:
We're just trying to standardize on [unix-y line endings](http://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html).